prereceivecli/lib/utils.py

```diff
@@ -189,12 +210,15 @@ def get_table_for_project_group(project_group):
 @contextmanager
 def no_quarantine():
     """Context manager that clears the GIT_QUARANTINE_PATH environment variable and restores it"""
+    git_quarantine_path = ''
     try:
-        git_quarantine_path = os.environ.get('GIT_QUARANTINE_PATH')
-        del os.environ['GIT_QUARANTINE_PATH']
-        yield
+        if os.environ.get('GIT_QUARANTINE_PATH'):
```

Comment:
Assign to variable, overwritting the variable set above and check instead of accessing the memory twice.

Reply:
you need to yield irrespective of whether the variable exists or not.